internal/pkg/daemon/bpfrecorder/bpfrecorderfakes/fake_impl.go

```diff
@@ -1,3 +1,19 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
```

Comment:
It complains about the go:build command:
```
# sigs.k8s.io/security-profiles-operator/internal/pkg/daemon/bpfrecorder/bpfrecorderfakes
internal/pkg/daemon/bpfrecorder/bpfrecorderfakes/fake_impl.go:18:1: misplaced +build comment
```

Reply:
ah... we might have an issue with `update-mocks` it seems